serve-react-app/Makefile

```
@@ -0,0 +1,13 @@
+assets := $(wildcard hello/*)
+
```

I think it'd be good here to have an install step for those who haven't yet installed Vite

```suggestion

install:
	npm install vite	
```

---

Better still

```suggestion

install: $(assets)
	npm --prefix hello/assets install
```

---

Very good idea, I wonder if the build command is even needed as well :thinking: